pkg/api/resource/quantity_test.go

```diff
@@ -57,6 +57,13 @@ func TestDec(t *testing.T) {
 	}
 }
 
+func TestQuantityParseZero(t *testing.T) {
+	zero := MustParse("0")
+	if expected, actual := "0", zero.String(); expected != actual {
```

Comment:
Yes.  Each of the tests I sent failed before the change. 

Sent from my iPhone

> On Jun 2, 2015, at 4:27 PM, Daniel Smith <notifications@github.com> wrote:
> 
> In pkg/api/resource/quantity_test.go:
> 
> > @@ -57,6 +57,13 @@ func TestDec(t *testing.T) {
> >  	}
> >  }
> >  
> > +func TestQuantityParseZero(t *testing.T) {
> > +	zero := MustParse("0")
> > +	if expected, actual := "0", zero.String(); expected != actual {
> Did this test fail before your change?
> 
> —
> Reply to this email directly or view it on GitHub.
>